projects/openapi-utilities/src/openapi3/types.ts

```diff
@@ -0,0 +1,115 @@
+import { OpenAPI, OpenAPIV3 } from 'openapi-types';
+
+export type V3FactType =
```

Comment:
@Alexandre-Herve This was the best i could get the types to - this (and the usage in `traverser + `group-diffs` is what i want to achieve with this. Any suggestions on how I can get away without some of the type casing in `getLocation` etc?

Reply:
No I don't know why it's not inferring that's odd 🤔 I'll try again later see if I can find something.